fuel-tx/src/lib.rs

```diff
@@ -125,6 +127,9 @@ pub use transaction::{
     Witness,
 };
 
+#[cfg(feature = "da-compression")]
+pub use transaction::CompressedTransaction;
```

Comment:
I think we need to export all types related to comression

Reply:
Do we? I would think the `fuel-core` part only operates on the transaction, and the types inside are never named.

Reply:
We don't need them. Just if we want to export `CompressedTransaction`, then it also could be useful to export other types to be consistent.

It can be useful during decompression to allows syntaxic like:

```rust
let ComressedCoin { utxod, predicate, ... } = value;
```

Reply:
I'm leaning towards exporting just the `CompressedTransaction`, and all types that do not derive `Decompress`